### PR TITLE
rocAL - Formatting and doxygen changes

### PR DESCRIPTION
**Pull request by SundarRajan28**
- Fixed doxygen warnings in amd_openvx and amd_openvx_extensions folders
- Added doxygen descriptors for rocAL API files
- Reformatted all C++ files using google coding style guide
- Added doxygen descriptors for rocAL python API files
- Reformatted all python files using autopep8 formatting tool
- Tested C++ and python examples after making the formatting and doxygen changes
Note: The previous PRs (https://github.com/LakshmiKumar23/MIVisionX/pull/35 , https://github.com/LakshmiKumar23/MIVisionX/pull/36, https://github.com/LakshmiKumar23/MIVisionX/pull/37 ) has to be merged before this PR.

### CODE REVIEW

File: amd_openvx_extensions/amd_custom/include/vx_amd_custom.h [hunk 1]
```diff
@@ -58,7 +54,7 @@ typedef struct _vx_amd_custom_params_t
  * \param [in] graph The handle to the graph.
  * \param [in] inputs The input tensor data.
  * \param [in] function custom funtion enum.
- * \param [in] array for user specified custom_parameters.
+ * \param [in] custom_parameters for user specified custom_parameters.
```
Comment (LakshmiKumar23) [inline amd_openvx_extensions/amd_custom/include/vx_amd_custom.h:57]: @fiona-gladwin @SundarRajan28 Please say custom_parameters <tt> vx_array </tt> for user specified... Needs to say what kind of input. Check on all files if any missing

File: amd_openvx_extensions/amd_nn/include/vx_amd_nn.h [hunk 2]
```diff
@@ -36,150 +36,139 @@ THE SOFTWARE.
...
  * \ingroup group_amd_nn
  * \param [in] graph The handle to the graph.
- * \param [in] inputs The input tensor data.
- * \param [in] inputs The mean tensor data.
- * \param [in] inputs The variance tensor data.
- * \param [in] inputs The scale tensor data.
- * \param [in] inputs The bias tensor data.
- * \param [in] inputs The eps vx_float32 data.
- * \param [out] outputs The output tensor data.
+ * \param [in] input The input tensor data.
```
Comment (LakshmiKumar23) [inline amd_openvx_extensions/amd_nn/include/vx_amd_nn.h:39]: @fiona-gladwin Why are we changing amd_nn files in a rocAL PR?

**Review by LakshmiKumar23 — CHANGES_REQUESTED** [review]
@fiona-gladwin This PR is changing amd_nn, amd_opencv, amd_winml. WHy is it in a rocAL PR?

**Comment by LakshmiKumar23** [comment]
@SundarRajan28 @fiona-gladwin can you look into the merge conflicts?

File: rocAL/rocAL/include/api/rocal_api_augmentation.h [hunk 3]
```diff
@@ -25,34 +25,38 @@ THE SOFTWARE.
...
+ * \brief Rearranges the order of the frames in the sequences with respect to new_order. new_order can have values in the range [0, sequence_length). Frames can be repeated or dropped in the new_order.
  * \ingroup group_rocal_augmentations
  * \note Accepts U8 and RGB24 input.
  * \param [in] p_context context for the pipeline.
  * \param [in] p_input Input Rocal Tensor
- * \param [in] new_order represents the new order of the frames in the sequence in the range [0, sequence_length)
- * \param [in] is_output True: the output tensor is needed by user and will be copied to output buffers using the data
- * transfer API calls. False: the output tensor is just an intermediate tensor, user is not interested in
- * using it directly. This option allows certain optimizations to be achieved.
+ * \param [in] new_order represents the new order of the frames in the sequence
```
Comment (LakshmiKumar23) [inline rocAL/rocAL/include/api/rocal_api_augmentation.h:41]: PLease mention "vector of type unsinged int epresenting"

**Review by LakshmiKumar23 — CHANGES_REQUESTED** [review]

**Comment by LakshmiKumar23** [comment]
@SundarRajan28 @fiona-gladwin I also do not see these files added to the [doxygen generator file](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/blob/master/.Doxyfile). Have you tried to build these files with DOxygen?

File: rocAL/rocAL/include/api/rocal_api_augmentation.h (hunk 3 above)
Comment (fiona-gladwin) [reply rocAL/rocAL/include/api/rocal_api_augmentation.h:41]: Done

**Comment by fiona-gladwin** [comment]
> @fiona-gladwin This PR is changing amd_nn, amd_opencv, amd_winml. WHy is it in a rocAL PR?

@LakshmiKumar23 While building Doxygen we saw that there were some warnings generated from the doxygen comments in these files. Hence we have fixed it in this PR

File: amd_openvx_extensions/amd_custom/include/vx_amd_custom.h (hunk 1 above)
Comment (fiona-gladwin) [reply amd_openvx_extensions/amd_custom/include/vx_amd_custom.h:57]: Done

**Comment by fiona-gladwin** [comment]
> @SundarRajan28 @fiona-gladwin I also do not see these files added to the [doxygen generator file](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/blob/master/.Doxyfile). Have you tried to build these files with DOxygen?

Done

File: rocAL/rocAL/include/augmentations/effects_augmentations/node_fog.h [hunk 4]
```diff
@@ -24,19 +24,18 @@ THE SOFTWARE.
 #include "node.h"
 #include "parameter_factory.h"
 #include "parameter_vx.h"
-class FogNode : public Node
-{
-public:
+class FogNode : public Node {
```
Comment (LakshmiKumar23) [inline rocAL/rocAL/include/augmentations/effects_augmentations/node_fog.h:28]: @fiona-gladwin Add a line space between 26 & 27. The other files seem to have that

File: rocAL/rocAL/include/decoders/image/decoder.h [hunk 5]
```diff
@@ -28,25 +28,22 @@ THE SOFTWARE.
...
-    OPENCV_DEC = 2, //!< for back_up decoding
-    HW_JPEG_DEC  = 3,
-    SKIP_DECODE  = 4, //!< For skipping decoding in case of uncompressed data from reader
-    OVX_FFMPEG = 5,//!< Uses FFMPEG to decode video streams, can decode up to 4 video streams simultaneously
+enum class DecoderType {
+    TURBO_JPEG = 0,        //!< Can only decode
+    FUSED_TURBO_JPEG = 1,  //!< FOR PARTIAL DECODING
+    OPENCV_DEC = 2,        //!< for back_up decoding
+    HW_JPEG_DEC = 3,
+    SKIP_DECODE = 4,  //!< For skipping decoding in case of uncompressed data from reader
```
Comment (LakshmiKumar23) [inline rocAL/rocAL/include/decoders/image/decoder.h]: @fiona-gladwin can you add space to align with comments on above lines

File: rocAL/rocAL/include/loaders/image/node_fused_jpeg_crop_single_shard.h [hunk 6]
```diff
@@ -43,10 +42,12 @@ class FusedJpegCropSingleShardNode: public Node
               unsigned num_attempts, std::vector<float> &random_area, std::vector<float> &random_aspect_ratio);
```
Comment (LakshmiKumar23) [inline rocAL/rocAL/include/loaders/image/node_fused_jpeg_crop_single_shard.h:42]: @fiona-gladwin  remove extra space

File: rocAL/rocAL/include/loaders/loader_module.h [hunk 7]
```diff
@@ -42,18 +42,17 @@ enum class LoaderModuleStatus
...
     virtual void initialize(ReaderConfig reader_config, DecoderConfig decoder_config, RocalMemType mem_type, unsigned batch_size, bool keep_orig_size) = 0;
     virtual void set_output(Tensor* output_tensor) = 0;
-    virtual LoaderModuleStatus load_next() = 0; // Loads the next image data into the Image's buffer set by calling into the set_output
-    virtual void reset() = 0; // Resets the loader to load from the beginning of the media
-    virtual size_t remaining_count() = 0; // Returns the number of available images to be loaded
-    virtual ~LoaderModule()= default;
-    virtual Timing timing() = 0;// Returns timing info
-    virtual std::vector<std::string> get_id() = 0; // returns the id of the last batch of images/frames loaded
-    virtual void start_loading() = 0; // starts internal loading thread
+    virtual LoaderModuleStatus load_next() = 0;  // Loads the next image data into the Image's buffer set by calling into the set_output
```
Comment (LakshmiKumar23) [inline rocAL/rocAL/include/loaders/loader_module.h]: @fiona-gladwin align all comments with spaces

File: rocAL/rocAL/include/augmentations/effects_augmentations/node_fog.h (hunk 4 above)
Comment (fiona-gladwin) [reply rocAL/rocAL/include/augmentations/effects_augmentations/node_fog.h:28]: Done

File: rocAL/rocAL/include/loaders/loader_module.h (hunk 7 above)
Comment (fiona-gladwin) [reply rocAL/rocAL/include/loaders/loader_module.h]: Done

File: rocAL/rocAL/include/decoders/image/decoder.h (hunk 5 above)
Comment (fiona-gladwin) [reply rocAL/rocAL/include/decoders/image/decoder.h]: Done

File: rocAL/rocAL/include/augmentations/effects_augmentations/node_fog.h (hunk 4 above)
Comment (fiona-gladwin) [reply rocAL/rocAL/include/augmentations/effects_augmentations/node_fog.h:28]: Done

File: rocAL/rocAL/include/loaders/image/node_fused_jpeg_crop_single_shard.h (hunk 6 above)
Comment (fiona-gladwin) [reply rocAL/rocAL/include/loaders/image/node_fused_jpeg_crop_single_shard.h:42]: @LakshmiKumar23 may I know where exactly to remove the extra space?

File: apps/image_augmentation/image_augmentation.cpp [hunk 8]
```diff
@@ -44,22 +42,21 @@ using namespace cv;
...
-    const char * folderPath1 = argv[++argIdx];
-    int video_mode = 0;// 0 means no video decode, 1 means hardware, 2 means software decoding
-    bool display = 1;// Display the images
-    int aug_depth = 1;// how deep is the augmentation tree
-    int rgb = 1;// process color images
+    const char* folderPath1 = argv[++argIdx];
+    int video_mode = 0;  // 0 means no video decode, 1 means hardware, 2 means software decoding
+    bool display = 1;    // Display the images
+    int aug_depth = 1;   // how deep is the augmentation tree
+    int rgb = 0;         // process color images
```
Comment (LakshmiKumar23) [inline apps/image_augmentation/image_augmentation.cpp]: @fiona-gladwin why is this changed?

File: rocAL/rocAL/include/pipeline/master_graph.h [hunk 9]
```diff
@@ -21,31 +21,32 @@ THE SOFTWARE.
...
 #if ENABLE_HIP
-#include "device_manager_hip.h"
 #include "box_encoder_hip.h"
+#include "device_manager_hip.h"
 #endif
 #include "randombboxcrop_meta_data_reader.h"
 #include "rocal_api_types.h"
 #define MAX_STRING_LENGTH 100
-#define MAX_OBJECTS 50 // Setting an arbitrary value 50.(Max number of objects/image in COCO dataset is 93)
+#define MAX_OBJECTS 50  // Setting an arbitrary value 50.(Max number of objects/image in COCO dataset is 93)
```
Comment (LakshmiKumar23) [inline rocAL/rocAL/include/pipeline/master_graph.h]: @fiona-gladwin Align with comment below

File: rocAL/rocAL/include/pipeline/master_graph.h [hunk 10]
```diff
@@ -140,18 +145,18 @@ class MasterGraph
     void notify_user_thread();
     /// no_more_processed_data() is logically linked to the notify_user_thread() and is used to tell the user they've already consumed all the processed tensors
     bool no_more_processed_data();
-    RingBuffer _ring_buffer;//!< The queue that keeps the tensors that have benn processed by the internal thread (_output_thread) asynchronous to the user's thread
-    pMetaDataBatch _augmented_meta_data = nullptr;//!< The output of the meta_data_graph,
+    RingBuffer _ring_buffer;                        //!< The queue that keeps the tensors that have benn processed by the internal thread (_output_thread) asynchronous to the user's thread
```
Comment (LakshmiKumar23) [inline rocAL/rocAL/include/pipeline/master_graph.h]: @fiona-gladwin Align with comments below. Please do that for this whole file

File: apps/image_augmentation/image_augmentation.cpp (hunk 8 above)
Comment (fiona-gladwin) [reply apps/image_augmentation/image_augmentation.cpp]: Changed to 1

File: rocAL/rocAL/include/pipeline/master_graph.h (hunk 9 above)
Comment (fiona-gladwin) [reply rocAL/rocAL/include/pipeline/master_graph.h]: Done

File: rocAL/rocAL/include/pipeline/master_graph.h (hunk 10 above)
Comment (fiona-gladwin) [reply rocAL/rocAL/include/pipeline/master_graph.h]: Done

**Comment by fiona-gladwin** [comment]
@LakshmiKumar23 resolved all review comments please check

File: rocAL/rocAL/source/api/rocal_api_meta_data.cpp [hunk 11]
```diff
@@ -410,91 +382,84 @@ ROCAL_API_CALL rocalGetSequenceStartFrameNumber(RocalContext p_context,  unsigne
...
+    for (unsigned int i = 0; i < sequence_frame_timestamps.size(); i++) {
         std::copy(sequence_frame_timestamps[i].begin(), sequence_frame_timestamps[i].end(), buf);
         buf = buf + sequence_length;
     }
 }
 
 void ROCAL_API_CALL rocalBoxEncoder(RocalContext p_context, std::vector<float>& anchors, float criteria,
-                                  std::vector<float> &means, std::vector<float> &stds, bool offset, float scale)
-{
+                                    std::vector<float>& means, std::vector<float>& stds, bool offset, float scale) {
```
Comment (LakshmiKumar23) [inline rocAL/rocAL/source/api/rocal_api_meta_data.cpp]: @fiona-gladwin please follow spacing and lines as other functions in this file

File: rocAL/rocAL/source/augmentations/geometry_augmentations/node_warp_affine.cpp [hunk 12]
```diff
@@ -60,23 +60,23 @@ void WarpAffineNode::create_node() {
...
+        _affine[i * 6 + 3] = _y1.renew();
+        _affine[i * 6 + 4] = _o0.renew();
+        _affine[i * 6 + 5] = _o1.renew();
     }
     vx_status affine_status;
-    affine_status = vxCopyArrayRange((vx_array)_affine_array, 0, _batch_size * 6, sizeof(vx_float32), _affine.data(), VX_WRITE_ONLY, VX_MEMORY_TYPE_HOST); //vxAddArrayItems(_width_array,_batch_size, _width, sizeof(vx_uint32));
-    if(affine_status != 0)
-        THROW(" vxCopyArrayRange failed in the WarpAffine(vxExtRppWarpAffinePD) node: "+ TOSTR(affine_status))
+    affine_status = vxCopyArrayRange((vx_array)_affine_array, 0, _batch_size * 6, sizeof(vx_float32), _affine.data(), VX_WRITE_ONLY, VX_MEMORY_TYPE_HOST);  // vxAddArrayItems(_width_array,_batch_size, _width, sizeof(vx_uint32));
+    if (affine_status != 0)
```
Comment (LakshmiKumar23) [inline rocAL/rocAL/source/augmentations/geometry_augmentations/node_warp_affine.cpp:78]: @fiona-gladwin remove this comment if not required

File: rocAL/rocAL/source/augmentations/node_copy.cpp [hunk 13]
```diff
@@ -20,21 +20,22 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
...
+#include "node_copy.h"
+
 #include <vx_ext_rpp.h>
 #include "node_copy.h"
 #include "exception.h"
 
-CopyNode::CopyNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs) :
-        Node(inputs, outputs) {}
+CopyNode::CopyNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs) : Node(inputs, outputs) {
+}
```
Comment (LakshmiKumar23) [inline rocAL/rocAL/source/augmentations/node_copy.cpp]: @fiona-gladwin move } one line up

Comment (fiona-gladwin) [reply rocAL/rocAL/source/augmentations/node_copy.cpp]: Done

File: rocAL/rocAL/source/augmentations/geometry_augmentations/node_warp_affine.cpp (hunk 12 above)
Comment (fiona-gladwin) [reply rocAL/rocAL/source/augmentations/geometry_augmentations/node_warp_affine.cpp:78]: Done

File: rocAL/rocAL/source/device/ocl_setup.cpp [hunk 14]
```diff
@@ -67,34 +64,30 @@ int get_device_and_context(int devIdx, cl_context *pContext, cl_device_id *pDevi
...
 
-
     // enumerate devices
 
     // To Do: handle multi-GPU case, pick appropriate GPU/APU
     char driverVersion[100] = "\0";
 
     cl_context_properties contextProps[3] =
-    {
+        {
```
Comment (LakshmiKumar23) [inline rocAL/rocAL/source/device/ocl_setup.cpp:87]: @fiona-gladwin move { one line up

File: rocAL/rocAL/source/loaders/video/video_loader.cpp [hunk 15]
```diff
@@ -41,33 +41,28 @@ _swap_handle_time("Swap_handle_time", DBG_TIMING)
...
+    if (_internal_thread_running)
         stop_internal_thread();
     _circ_buff.release();
 }
 
-void VideoLoader::set_prefetch_queue_depth(size_t prefetch_queue_depth)
-{
+void VideoLoader::set_prefetch_queue_depth(size_t prefetch_queue_depth) {
     if (prefetch_queue_depth <= 0)
         THROW("Prefetch quque depth value cannot be zero or negative");
```
Comment (LakshmiKumar23) [inline rocAL/rocAL/source/loaders/video/video_loader.cpp]: @fiona-gladwin 
spell check "queue"

File: rocAL/rocAL/source/meta_data/bounding_box_graph.cpp [hunk 16]
```diff
@@ -149,101 +134,86 @@ inline void calculate_ious_for_box(float *ious, BoundingBoxCord &box, BoundingBo
...
+                    box_bestidx.xcycwh.yc = 0.5 * (bb_coords[best_idx].t + bb_coords[best_idx].b);  // yc
+                    box_bestidx.xcycwh.w = bb_coords[best_idx].r - bb_coords[best_idx].l;           // w
+                    box_bestidx.xcycwh.h = bb_coords[best_idx].b - bb_coords[best_idx].t;           // h
                     encoded_bb[anchor_idx] = box_bestidx;
                     encoded_labels[anchor_idx] = bb_labels[best_idx];
                 }
-            }
-            else // Not a match
+            } else  // Not a match
             {
```
Comment (LakshmiKumar23) [inline rocAL/rocAL/source/meta_data/bounding_box_graph.cpp]: @fiona-gladwin move { one line up

File: rocAL/rocAL/source/meta_data/tf_meta_data_reader.cpp [hunk 17]
```diff
@@ -21,99 +21,92 @@ THE SOFTWARE.
...
-    if(!file_contents)
+    file_contents.read(data, data_length);
+    if (!file_contents)
         THROW("TFMetaDataReader: Error in reading TF records")
     tensorflow::Example single_example;
-    single_example.ParseFromArray(data,data_length);
+    single_example.ParseFromArray(data, data_length);
     tensorflow::Features features = single_example.features();
-//..............
+    //..............
```
Comment (LakshmiKumar23) [inline rocAL/rocAL/source/meta_data/tf_meta_data_reader.cpp]: @fiona-gladwin what is this comment format? Please rmeove unnecessary ones

File: rocAL/rocAL/source/readers/image/cifar10_data_reader.cpp [hunk 18]
```diff
@@ -61,132 +60,118 @@ Reader::Status CIFAR10DataReader::initialize(ReaderConfig desc)
...
     _last_id.append(std::to_string(_last_file_idx));
     // compare the file_name with the last one opened
-    if ( file_path.compare(_last_file_name) != 0) {
+    if (file_path.compare(_last_file_name) != 0) {
         if (_current_fPtr) {
             fclose(_current_fPtr);
             _current_fPtr = nullptr;
         }
-        _current_fPtr = fopen(file_path.c_str(), "rb");// Open the file,
+        _current_fPtr = fopen(file_path.c_str(), "rb");  // Open the file,
```
Comment (LakshmiKumar23) [inline rocAL/rocAL/source/readers/image/cifar10_data_reader.cpp:88]: @fiona-gladwin please align all comments in this file

File: rocAL/rocAL_pybind/amd/rocal/plugin/pytorch.py [hunk 19]
```diff
@@ -98,32 +124,41 @@ def __next__(self):
                         img = output
                         draw_patches(img[i], i, self.bb_2d_numpy)
 
-
-            max_cols = max([len(row) for batch in self.bbox_list for row in batch])
+            max_cols = max([len(row)
```
Comment (LakshmiKumar23) [inline rocAL/rocAL_pybind/amd/rocal/plugin/pytorch.py:126]: @fiona-gladwin why add the extra line?

File: rocAL/rocAL_pybind/amd/rocal/plugin/tf.py [hunk 20]
```diff
@@ -121,26 +145,33 @@ def __next__(self):
...
             self.target1 = self.label_list
-            max_cols = max([len(row) for batch in self.target for row in batch])
+            max_cols = max([len(row)
+                           for batch in self.target for row in batch])
             # max_rows = max([len(batch) for batch in self.target])
             max_rows = 100
-            bb_padded = [batch + [[0] * (max_cols)] * (max_rows - len(batch)) for batch in self.target]
-            bb_padded_1 = [row + [0] * (max_cols - len(row)) for batch in bb_padded for row in batch]
+            bb_padded = [
+                batch + [[0] * (max_cols)] * (max_rows - len(batch)) for batch in self.target]
```
Comment (LakshmiKumar23) [inline rocAL/rocAL_pybind/amd/rocal/plugin/tf.py:162]: @fiona-gladwin why have these extra lines been added?

File: rocAL/rocAL_pybind/examples/rocAL_api_caffe_reader.py [hunk 21]
```diff
@@ -121,9 +127,11 @@ def main():
                         print("Labels", labels)
                 for element in list(range(batch_size)):
                     cnt += 1
-                    draw_patches(image_batch[element], cnt, bboxes[element], args=args)
+                    draw_patches(image_batch[element],
+                                 cnt, bboxes[element], args=args)
             data_loader.reset()
     print("##############################  CAFFE READER (CLASSIFCATION/ DETECTION)  SUCCESS  ############################")
 
+
```
Comment (LakshmiKumar23) [inline rocAL/rocAL_pybind/examples/rocAL_api_caffe_reader.py]: @fiona-gladwin remove the extra line

File: rocAL/rocAL_pybind/rocal_pybind.cpp [hunk 22]
```diff
@@ -38,567 +38,553 @@ namespace py = pybind11;
...
-            static pybind11::dtype dtype()
-            {
-                handle ptr = npy_api::get().PyArray_DescrFromType_(NPY_FLOAT16);
-                return reinterpret_borrow<pybind11::dtype>(ptr);
-            }
-        };
+namespace pybind11 {
+namespace detail {
+constexpr int NPY_FLOAT16 = 23;
+// Kinda following: https://github.com/pybind/pybind11/blob/9bb3313162c0b856125e481ceece9d8faa567716/include/pybind11/numpy.h#L1000
```
Comment (LakshmiKumar23) [inline rocAL/rocAL_pybind/rocal_pybind.cpp]: @fiona-gladwin Please change language . what does this comment even mean? Remove if not necessary

File: utilities/rocAL/rocAL_unittests/rocAL_unittests.cpp [hunk 23]
```diff
@@ -808,29 +672,27 @@ int test(int test_case, int reader_type, const char *path, const char *outName,
...
         if (rocalRun(handle) != 0)
             break;
         int numOfClasses = 0;
         int image_name_length[inputBatchSize];
-        switch(pipeline_type)
-        {
-            case 1: //classification pipeline
+        switch (pipeline_type) {
+            case 1:  // classification pipeline
             {
```
Comment (LakshmiKumar23) [inline utilities/rocAL/rocAL_unittests/rocAL_unittests.cpp]: @fiona-gladwin move { one line up. For all cases in this switch statement

File: rocAL/rocAL/source/device/ocl_setup.cpp (hunk 14 above)
Comment (fiona-gladwin) [reply rocAL/rocAL/source/device/ocl_setup.cpp:87]: The C++ formatter resulted in this kind of formatting. Since it is not a function definition

File: rocAL/rocAL_pybind/amd/rocal/plugin/pytorch.py (hunk 19 above)
Comment (fiona-gladwin) [reply rocAL/rocAL_pybind/amd/rocal/plugin/pytorch.py:126]: The line is removed and line 126 is present to separate from the for loop block

File: rocAL/rocAL/source/loaders/video/video_loader.cpp (hunk 15 above)
Comment (fiona-gladwin) [reply rocAL/rocAL/source/loaders/video/video_loader.cpp]: Done

File: rocAL/rocAL/source/meta_data/bounding_box_graph.cpp (hunk 16 above)
Comment (fiona-gladwin) [reply rocAL/rocAL/source/meta_data/bounding_box_graph.cpp]: Done

File: rocAL/rocAL/source/meta_data/tf_meta_data_reader.cpp (hunk 17 above)
Comment (fiona-gladwin) [reply rocAL/rocAL/source/meta_data/tf_meta_data_reader.cpp]: Done

File: rocAL/rocAL_pybind/amd/rocal/plugin/tf.py (hunk 20 above)
Comment (fiona-gladwin) [reply rocAL/rocAL_pybind/amd/rocal/plugin/tf.py:162]: The python formatter resulted in this kind of formatting, where it split longer lined into 2 lines

File: rocAL/rocAL_pybind/examples/rocAL_api_caffe_reader.py (hunk 21 above)
Comment (fiona-gladwin) [reply rocAL/rocAL_pybind/examples/rocAL_api_caffe_reader.py]: Done

File: utilities/rocAL/rocAL_unittests/rocAL_unittests.cpp (hunk 23 above)
Comment (fiona-gladwin) [reply utilities/rocAL/rocAL_unittests/rocAL_unittests.cpp]: Done

File: rocAL/rocAL_pybind/rocal_pybind.cpp (hunk 22 above)
Comment (fiona-gladwin) [reply rocAL/rocAL_pybind/rocal_pybind.cpp]: Changed the words

File: rocAL/rocAL/source/readers/image/cifar10_data_reader.cpp (hunk 18 above)
Comment (fiona-gladwin) [reply rocAL/rocAL/source/readers/image/cifar10_data_reader.cpp:88]: Used the C++ formatter in the whole file

File: rocAL/rocAL/source/api/rocal_api_meta_data.cpp (hunk 11 above)
Comment (fiona-gladwin) [reply rocAL/rocAL/source/api/rocal_api_meta_data.cpp]: Done

**Comment by fiona-gladwin** [comment]
@LakshmiKumar23 I have addressed all review comments

File: rocAL/rocAL_pybind/rocal_pybind.cpp [hunk 24]
```diff
@@ -41,7 +41,7 @@ static_assert(sizeof(float16) == 2, "Bad size");
 namespace pybind11 {
 namespace detail {
 constexpr int NPY_FLOAT16 = 23;
-// Kinda following: https://github.com/pybind/pybind11/blob/9bb3313162c0b856125e481ceece9d8faa567716/include/pybind11/numpy.h#L1000
+// Similar to the implementation in: https://github.com/pybind/pybind11/blob/9bb3313162c0b856125e481ceece9d8faa567716/include/pybind11/numpy.h#L1000
```
Comment (LakshmiKumar23) [inline rocAL/rocAL_pybind/rocal_pybind.cpp]: @fiona-gladwin why do we have to mention this at all?

Comment (fiona-gladwin) [reply rocAL/rocAL_pybind/rocal_pybind.cpp]: Removed the line

**Review by LakshmiKumar23 — APPROVED** [review]
